eth/api_backend.go

```diff
@@ -323,6 +324,31 @@ func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction)
 	return b.eth.txPool.Add([]*types.Transaction{signedTx}, true, false)[0]
 }
 
+func (b *EthAPIBackend) SendBundle(ctx context.Context, bundle *types.Bundle, originBundle *types.SendBundleArgs) error {
+	return b.eth.txPool.AddBundle(bundle, originBundle)
+}
+
+func (b *EthAPIBackend) BundlePrice() *big.Int {
+	bundles := b.eth.txPool.AllBundles()
+	gasFloor := big.NewInt(b.eth.config.Miner.Mev.MevGasPriceFloor)
+
+	if len(bundles) == 0 {
+		return gasFloor
+	}
+
+	sort.SliceStable(bundles, func(i, j int) bool {
+		return bundles[j].Price.Cmp(bundles[i].Price) < 0
+	})
+
+	idx := len(bundles) / 2
```

Comment:
In fact, I have removed most of the useless configurations. For this, I think taking the median gasprice as return value is reasonable. I don’t expect it to be changed anymore. The second thing is that I think if this value is configurable , it is the node behavior, it will make the return price logic unstable and confusing.